### PR TITLE
Paid model Completion

### PR DESCRIPTION
**Pull request by OAGr**
Several touches:
- Updated FAQ to contain information on private models
- Improved design for payment flow
- Fixed bug regarding clicking on the customer portal multiple times
- New upgrade button in account settings menu
- All users can officially see payment options
- Replaced several font awesome icons with Ionicons

### CODE REVIEW

File: src/components/pages/faq/index.js [hunk 1]
```diff
@@ -11,13 +11,13 @@ Guesstimate is great at making estimates of things that aren't certain.  You can
 The most popular model on Guesstimate is [How Long it Takes to Get Ready for Preschool](/models/314).  We also recommend checking out Ozzie Gooen's models [here](/users/1).
 
 ## Are there private models?
-Not yet.  Please sign up for an account to get on our mailing list, or follow us [on Twitter](https://twitter.com/getguesstimate) to be notified once there are.  We expect private models in the next 1-6 weeks.
+Yes. Check out our [pricing page](/pricing) for more information.
```
Comment (mmcdermott) [inline src/components/pages/faq/index.js]: Yes. -> Yes!

At the end say 'Let us know what other features you want in our premium accounts.'

File: src/components/spaces/show/header.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ const SpaceHeader = ({canMakeMorePrivateModels, space, isLoggedIn, onSave, onCop
           </DropDown>
         }
 
-        {space.ownedByMe && canUsePrivateModels &&
+        {space.ownedByMe &&
           <PrivacyToggleDropdown
```
Comment (mmcdermott) [inline src/components/spaces/show/header.js]: We need to make the 'upgrade your account' link to the portal/pricing page.

File: src/components/subscriptions/FirstSubscription/FirstSubscription.js [hunk 3]
```diff
@@ -81,14 +89,35 @@ export const FormSuccess = ({iframeUrl, iframeWebsiteName, onPaymentCancel, onPa
...
   </div>
 )
 
-export const Cancelled = () => ( <div> Payment Cancelled.  Refresh to try again. </div> )
-export const FormStart = () => ( <div> Loading... </div> )
-export const FormFailure = () => ( <div> The form failed loading.  Try again soon. </div> )
-export const SynchronizationStart = () => ( <div> Synchronizing... </div> )
-export const SynchronizationSuccess = () => ( <div> Payment Complete. </div> )
-export const SynchronizationFailure = () => ( <div> Synchronization Failed.  You have paid.  Contact Ozzie if there are issues with this. </div> )
+export const Cancelled = () => (
```
Comment (mmcdermott) [inline src/components/subscriptions/FirstSubscription/FirstSubscription.js]: Remove this (is dead code afaik)

File: src/components/pages/faq/index.js (hunk 1 above)
Comment (OAGr) [inline src/components/pages/faq/index.js]: I'm not a huge fan of exclamation marks.  The 'let us know' seems weird for the specific question 'are there private models'.

File: src/components/spaces/show/header.js (hunk 2 above)
Comment (OAGr) [inline src/components/spaces/show/header.js]: True, will be a separate story.

**Comment by OAGr** [comment]
Merging now, fix will come later.

**Pull request merged**